config/dev.json

```diff
@@ -1,22 +1,45 @@
 {
-  "port":9000,
+  "environment": "production",
+  "https": {
+    "enabled": false
+  },
   "logger": {
-    "name": "logger",
-    "streams": [{
-      "type": "stream",
-      "src": true,
-      "level": "trace",
-      "stream": "process.stdout"
-    }, {
-      "type": "raw",
-      "src": true,
-      "level": "trace",
-      "stream": "ringBuffer"
-    }]
+    "name": "dataman",
+    "streams": [
+      {
+        "type": "file",
+        "stream": "file",
+        "path": "/var/log/feedhenry/fh-dataman/fh-dataman.log",
+        "level": "info"
+      },
+      {
+        "type": "raw",
+        "src": true,
+        "level": "info",
+        "colorize": true
+      }
+    ]
   },
   "auth": {
-    "secret": "secret"
+    "secret": "feedhenry101"
   },
   "businessObject": "databrowser",
-  "sizeLimit": 524288000
+  "port": 8822,
+  "sizeLimit": 524288000,
+  "rootUrl": "/",
+  "mbaas": {
+    "mbaasUrl": "node1.feedhenry.local",
+    "password": "feedhenry101",
+    "username": "feedhenry",
+    "service_key": "mBqxpulwlj3Rw7EsHZq0YEQyrArq2tlIRCQ0pCmr5DRPPDZMAP4ZNb7U0E6bPSb3"
+  },
+  "ditch": {
+    "host": "node1.feedhenry.local",
+    "port": 8802,
+    "protocol": "http",
```

Comment:
The protocol will always be `mongodb` (https://github.com/feedhenry/fh-dataman/blob/test-integration-proxy-debugging/src/middleware/dbConnection/index.js#L38) so I don't think we need it here?